### DISCUSSION TITLE
init docs content into a usb memory

### DISCUSSION THREAD

**Discussion by jpochguardado** [tighten/jigsaw#645] [Q&A] [answered]
I'm testing a 'init docs' site for saving it int a usb memory in order to access in this way.
But when I access from within the usb memory many url go wrong and the site does not see fine.
Is there any way for doing this?
Thanks!

**Comment by bakerkretzmar** [comment]
Can you share more information about what you're trying to do? What are the incorrect URLs and what do you expect them to be?

I think you should be able to develop and/or view a Jigsaw site on a USB key normally, same as you would do it anywhere else. It's all just static files.

**Comment by jpochguardado** [comment]
All this urls in index.php:

    <link rel="stylesheet" href="/assets/build/css/main.css">
    <img src="/assets/img/logo-large.svg" alt="{{ $page->siteName }} large logo" class="mx-auto mb-6 lg:mb-0 ">
    <img src="/assets/img/icon-window.svg" class="h-12 w-12" alt="window icon">

do a wrong access when viewing the contents not through a webserver.

For now, I have installed a Portable Webserver.

Thanks

**Reply by bakerkretzmar** [reply]
What do you mean they "do a wrong access"? What is the actual error message?

Jigsaw doesn't need a web server and your site output shouldn't have PHP files in it by default. It should be plain HTML and you should be able to open the files in any browser.

**Reply by husseinalhammad** [reply]
Assuming your site is not in the root folder of the USB memory, you probably need to update the paths.

Instead of always using `/assets/etc/`, you may need to use relative paths like `assets/etc/`, `../assets/etc` or `../../assets/etc`  (and so on) based on where the page is in your site's structure.

**Comment by jpochguardado** [comment, marked as answer]
With "do a wrong access" I mean the urls are incorrectly formed. yes my site output doesn't have PHP files. I can open in a browser but the urls are wrong, so the site does not see correctly.
Yes is a urls repair stuff.

Thank you!
